Lombiq.ChartJs.Samples/Assets/Styles/chartjs-samples.scss

```diff
@@ -0,0 +1,5 @@
+.lombiqChartJsSamplesChartContainer {
```

Comment:
Commit the output from wwwroot as well, [see here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/#:~:text=for%20gulp%20extension-using%20projects%20you'll%20need%20to%20commit%20the%20wwwroot%20folder%20for%20now%2C%20see%20this%20issue.).